java/arcs/sdk/android/storage/service/StorageService.kt

```diff
@@ -165,7 +165,9 @@ open class StorageService : ResurrectorService() {
         return BindingContext(
             stores.computeIfAbsent(options.storageKey) {
                 @Suppress("UNCHECKED_CAST")
-                DeferredStore<CrdtData, CrdtOperation, Any>(options)
+                DeferredStore<CrdtData, CrdtOperation, Any>(
+                    options.copy(coroutineScope = CoroutineScope(coroutineContext))
```

Comment:
Let's create a single scope owned by StorageService and pass that in here, rather than creating a new one every time.

Reply:
It's a little tricky here.
AiAiStorageService extends StorageService and overrides the property `coroutineContext`.
So we cannot create a single scope based on `coroutineContext` during the class's ctor/init due to base/children ctor/init order.

Reply:
Ah, yeah

I think we should eventually fix the base class to define a scope, and provide it from a method explicitly to help with this...

Is there any problem with using lazy init for the scope?

Reply:
No problem with lazy init, I can change it.